src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/SortableColumnHeader.cs

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+using System.ComponentModel;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Documents;
+
+namespace NuGet.PackageManagement.UI
+{
+    public class SortableColumnHeader
+    {
+        public static string GetSortPropertyName(DependencyObject obj)
+        {
```

Comment:
validate for null here and in every other new method.
Tip: we've adopted `Assumes` API for pre-requisite validations, so here it'd be `Assumes.Present(obj);`. `Present` gives a better message than `Assumes.NotNull`.